tests/test_radius_server.rb

```diff
@@ -47,21 +54,37 @@ def test_create_destroy_single
     id = '8.8.8.8'
     refute_includes(Cisco::RadiusServer.radiusservers, id)
 
-    server = Cisco::RadiusServer.new(id, true)
+    # The 55 and 44 are the acct and auth ports respectively for EXR
+    # They should be ignored for other platforms with the values remaining
+    # as the defaults, which is part of this test case.
```

Comment:
I still don't see the benefit of this approach. Why not instead have Nexus also call auth_port = 44 and acct_port = 55 if the values are provided in the constructor?